browser/gui/app.cpp

```diff
@@ -123,6 +123,12 @@ App::App(std::string browser_title, std::string start_page_hint, bool load_start
         spdlog::critical("imgui-sfml initialization failed");
         std::abort();
     }
+    // ImGui tries to write to the current working directory. This can be problematic for a number of reasons,
+    // so it can be prevented via an env var.
+    if (std::getenv("HASTUR_FORBID_DISKIO")) {
+        // TODO we need a wrapper to support bool & string env vars
```

Comment:
Why? `std::getenv` returns a `char *` which you can wrap in a `string_view` without any allocations or anything if you need a string.

Reply:
Based on the assumption that eventually we'd like to support a number of env settings where the string may be one of `"0", "1", "2", "true", "FALSE", "T", "false "` etc. and it'd be user hostile to only permit one flavor of the above, or do the wrong thing when someone tries to debug and changes `HST_FORBID_IO_DISK=0" to disable forbidding, or enable IO.